data/model/gc.py

```diff
@@ -201,6 +201,30 @@ def _purge_repository_contents(repo):
         if not found:
             break
 
+    # Purge manifests that not referenced by tags
+    while True:
+        found = False
+        for manifests in _chunk_iterate_for_deletion(
+            Manifest.select().where(Manifest.repository == repo)
```

Comment:
can we send this query to the read replica?

Reply:
For GC I would say that we want to make sure that we have the up-to-date data